main.py

```diff
@@ -185,3 +188,31 @@ def get_cookie(dpid):
         """Get 8-byte integer cookie."""
         int_dpid = int(dpid.replace(":", ""), 16)
         return (0x00FFFFFFFFFFFFFF & int_dpid) | (settings.COOKIE_PREFIX << 56)
+
+    def set_flow_table_group_owner(self, flow: dict, group: str) -> dict:
+        """Set owner, table_group and table_id
+        coloring is only allowing 'base' for now"""
+        flow["owner"] = "coloring"
+        flow["table_group"] = group
+
+        # If "base" in table_group, every flow is in one table
+        if "base" in self.table_group:
```

Comment:
This conditional looks good, but at this point here, at the application core level, the expected table_group should be there, and if the caller pass a wrong one, since this code is only called from a trusted part then it should crash. So, the lookup should be `self.table_group["base"]`, and then validations you can do on `on_table_enabled(self, event)`, there you can have a minimal sanity that only set the table group if it has a well-known supported table group, and if it doesn't' you can log as error, we don't need other more elaborated error handling in this part specifically as we discussed in the blueprint, but if an accidental mistake is made on `of_multi_table` when sending `"kytos/of_multi_table.enable_table"` months/years down the road, then at this handler at least it should log it. Can you update these parts accordingly?

Reply:
Added `log.error()` to both methods. `set_flow_table_group_owner()` will crash for unknown table group and `on_table_enabled()` will search through the event for any not allowed table group defined in settings `TABLE_GROUP_ALLOWED = {"base"}`

Reply:
Nice, great to have that on settings. No need for `log.error` on both methods though (https://github.com/kytos-ng/coloring/pull/44#discussion_r1178358796)